system-int-test/test/cmr/system_int_test/search/granule_counts_search_test.clj

```diff
@@ -426,4 +432,20 @@
                      (search/find-refs :collection
                                        {:has_granules_or_cwic false
                                         :page-size 20}
-                                       {:snake-kebab? false})))))
+                                       {:snake-kebab? false})))
+
+    (testing "Sorting by has-granules-or-cwic"
```

Comment:
It wasn't obvious to me that this is the right order. Is there another test that sorts by has-granules that has a different order than this one because there are some cwic collections?

Reply:
This file has the sort test for has-granules.  I can write a dedicated sort test for has-granules-or-cwic to make it more obvious if you'd like.